src/main/java/org/spongepowered/api/world/gen/PopulatorObject.java

```diff
@@ -28,7 +28,10 @@
 import org.spongepowered.api.world.World;
 
 /**
- * Represents a structure which can be placed into the world by a populator.
+ * Represents a relatively small object (must fit in a single chunk) that can be
```

Comment:
This is worded poorly. I meant that the object may not be larger than 16 x 256 x 16 blocks, not that the object cannot cross chunk boundaries.

Placing larger objects requires some careful planning to avoid hitting unloaded chunks, which is why Minecraft's structure code is so complex. Structures are not part of this PR, but I can imagine a Structure interface that holds many smaller PopulatorObject.